lib/core/collapsable/accordion-group.component.spec.ts

```diff
@@ -115,4 +115,15 @@ describe('AccordionGroupComponent', () => {
         header.click();
     }));
 
+    it('should expand if panel is not open and is clicked', () => {
+        component.isSelected = true;
+        component.isOpen = false;
```

Comment:
you have removed the condition related to the isOpen
```javascript
-    isExpandable(event: any) {
-        if (!this.hasContent || !this.isOpen) {
+    isExpandable() {
+        if (!this.hasContent) {
             this.expandPanel();
         }
     }
```
so from my point of view the test should be
it('should expand if panel has content and is clicked'

Create a custom component that **contains an accordion with a content**
like https://github.com/Alfresco/alfresco-ng2-components/blob/dee4b0f57b4b97c7de9ec69ab0dad782a53a05fb/lib/core/info-drawer/info-drawer.component.spec.ts#L83



it('should not expand if panel doens't have content and is clicked'
Create a custom component that **contains an accordion without a content**